lib/themes/makeTreatTheme.ts

```diff
@@ -45,6 +65,16 @@ const decorateTheme = (vars: ThemeVars) => ({
       infoContrast: getAccessibleVariant(vars.color.foreground.info),
       positiveContrast: getAccessibleVariant(vars.color.foreground.positive),
     },
+    background: {
+      ...vars.color.background,
+      formAccentActive: getActiveColor(vars.color.background.formAccent),
+      formAccentHover: getHoverColor(vars.color.background.formAccent),
+      brandAccentActive: getActiveColor(vars.color.background.brandAccent),
+      brandAccentHover: getHoverColor(vars.color.background.brandAccent),
+      infoLight: getLightVariant(vars.color.background.info),
+      criticalLight: getLightVariant(vars.color.background.critical),
+      positiveLight: getLightVariant(vars.color.background.positive),
```

Comment:
I wonder if we should put these values on the theme or just the util itself and calculate where it's needed?

Reply:
Hmm, i like the thinking, let me see what the implications are.